ons_ras_common/ons_environment.py

```diff
@@ -68,6 +75,9 @@ def __init__(self):
         self._jwt = ONSJwt(self)
         self._cryptography = ONSCryptographer(self)
         self._registration = ONSRegistration(self)
+        self._asyncio = ONSAsyncIO(self)
```

Comment:
I'd suggest this class is increasingly in need of refactoring. As far as I can tell, it has unclear responsibilities, and cohesion is lower than it should be. It would be useful to apply SOLID design principles, for example to carry out dependency injection rather than hard-code a fixed set of dependencies.